ui/app/src/components/PathNavigatorTree/PathNavigatorTree.tsx

```diff
@@ -210,14 +210,11 @@ export function PathNavigatorTree(props: PathNavigatorTreeProps) {
   useEffect(() => {
     const subscription = onSearch$.pipe(debounceTime(400)).subscribe(({ keyword, path }) => {
       dispatch(
-        batchActions([
-          pathNavigatorTreeSetKeyword({
-            id,
-            path,
-            keyword
-          }),
-          pathNavigatorTreeBackgroundRefresh({ id })
-        ])
+        pathNavigatorTreeSetKeyword({
```

Comment:
pathNavigatorTreeBackgroundRefresh was added in [here](https://github.com/craftercms/studio-ui/pull/2891/commits/efe643228e0f06c4767ac968bc24c02303d625d7?show-viewed-files=true&file-filters%5B%5D=#diff-7795435422e5856c2f6e960204ac5212e622b060458ef6f8786945f87236f69fR260) after removing `resetBackgroundRefreshInterval`. Which seems incorrect as it only was reseting the interval, not dispatching the action. That was causing most of the issues related with this ticket.